### PR TITLE
Shenli/show variables

### PR DESCRIPTION
**Pull request by shenli**
Fix an error when run phpMyAdmin
TODO: support where in show variables

### CODE REVIEW

File: plan/plans/from_test.go [hunk 1]
```diff
@@ -69,11 +69,9 @@ func (p *testFromSuit) ClearValue(key fmt.Stringer) {}
 
 var _ = Suite(&testFromSuit{})
 
-var store kv.Storage
-
 func (p *testFromSuit) SetUpSuite(c *C) {
 	var err error
```
Comment (zimulala) [inline plan/plans/from_test.go]: This line can be removed

Comment (shenli) [inline plan/plans/from_test.go]: yes

File: plan/plans/show_test.go [hunk 2]
```diff
@@ -11,4 +11,103 @@
...
+	"github.com/pingcap/tidb/stmt"
+)
+
+type testShowSuit struct {
+	txn  kv.Transaction
+	vars map[string]interface{}
+}
+
+// implement Context interface
+func (p *testShowSuit) GetTxn(forceNew bool) (kv.Transaction, error) { return p.txn, nil }
```
Comment (siddontang) [inline plan/plans/show_test.go]: you can use a mock Context in util/mock

Comment (shenli) [inline plan/plans/show_test.go]: For show statement, we need to access storage and will encounter error if use the mock context.

Comment (siddontang) [inline plan/plans/show_test.go]: Got it.

**Comment by shenli** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by ngaut** [comment]
LGTM

**Pull request merged**